README.md

```diff
@@ -97,6 +97,7 @@ The following APIs are shared by Slider and Range.
 | min | number | `0` | The minimum value of the slider |
 | max | number | `100` | The maximum value of the slider |
 | marks | `{number: string}` or`{number: { style, label }}` | `{}` | Marks on the slider. The key determines the position, and the value determines what will show. If you want to set the style of a specific mark point, the value should be an object which contains `style` and `label` properties. |
+| markFontSize | number | 10 | The font size used for the mark labels. This is used to calculate the width of the mark labels. |
```

Comment:
I don't think this is a good idea, the actual font size is out of control.

Reply:
I don't know how else we can accurately calculate the width of a mark. Any suggestions?